framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2218,6 +2218,30 @@ protected boolean testTypevarContainment(
     }
 
     /**
+     * Create an OverrideChecker.
+     *
+     * <p>This exists so that subclasses can subclass OverrideChecker and use their subclass instead
+     * of using OverrideChecker itself.
+     */
+    protected OverrideChecker makeOverrideChecker(
```

Comment:
I would rename this method to `createOverrideChecker` to be in line with the other `createXXX` methods we have.